meta/runtime.yml

```
@@ -708,6 +708,8 @@ plugin_routing:
       redirect: community.general.remote_management.manageiq.manageiq_policies
     manageiq_provider:
       redirect: community.general.remote_management.manageiq.manageiq_provider
+    manageiq_provider_info:
+      redirect: community.general.remote_management.manageiq.manageiq_provider_info
```

This needs to be moved to the correct place:
```suggestion
      redirect: community.general.remote_management.manageiq.manageiq_policies_info
```

---

geez, I wasn't really paying much attention was I? (facepalm)